src/modules/addRemove.js

```diff
@@ -0,0 +1,174 @@
+import Task from './task.js';
+
+class displayTask {
+  static displayTasks() {
+    const tasks = Store.getTasks();
+    tasks.forEach((task) => displayTask.addTaskToList(task));
+  }
+
+  static addTaskToList(task) {
+    const elementList = document.querySelector('.list');
+    const listItem = document.createElement('li');
+    listItem.setAttribute('class', 'item');
+
+    listItem.innerHTML = `<input type="checkbox" id='${task.id}' name="taskStatus" class = 'taskStatus'>
+                    <p class='description edit' contenteditable='true' id='${task.id}'>${task.description}</p>
+                    <i class="fas fa-trash-alt remove-btn" id='${task.id}'></i>`;
+
+    elementList.appendChild(listItem);
+  }
+
+  static deleteTask(el) {
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#inputTask').value = '';
+  }
+}
+
+class Store {
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+
+    return tasks;
+  }
+
+  static addTask(task) {
+    const tasks = Store.getTasks();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static removeTask(elem) {
+    if (elem.classList.contains('remove-btn')) {
+      const tasks = Store.getTasks();
+      if (tasks.length === 1) {
+        tasks.pop();
+      } else {
+        tasks.splice(parseInt(elem.id), 1);
+
+        tasks.forEach((element, index) => {
+          element.id = index + 1;
+        });
+      }
+      index = tasks.length;
+
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
```

Comment:
- [ ] In continuation of the edit problem above, your deletion does not work as expected. I deleted the `first task`, yet is shows up again in the storage, so that when I reload it is still there. Kindly fix this behavior. Only the proper task should be deleted.

  "First task" is deleted | "First task" still shows up while "first tas" disappears
  :---:|:---:
  ![shing-del-1](https://user-images.githubusercontent.com/36542986/165675710-5f88606a-d1fd-45a8-9d71-52e52b427c4d.jpg) | ![shing-del-2](https://user-images.githubusercontent.com/36542986/165675712-69a3cd39-9f58-4f8f-a60c-b76ea83aef05.jpg)